tests/flow/test_udf.py

```
@@ -0,0 +1,1077 @@
+        self._assert_udf_missing("persist_del_lib")
+
+        # restart server
+        self.env.restart_and_reload()
+
+        # confirm library did not come back
+        self._assert_udf_missing("persist_del_lib")
+
+        # confirm removed function isn't callable
+        try:
+            self.graph.query ("RETURN persist_del_lib.PersistDel()")
+            assert False, f"Expected failure calling deleted function {f}"
```

Ruff (0.14.7)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)